clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -1308,6 +1309,60 @@ export const CustomDatePicker = ({
           size="sm"
           focusBorderColor="primary.600"
           data-testid={`input-${name}`}
+          disabled={isDisabled}
+        />
+        <ErrorMessage
+          isInvalid={isInvalid}
+          message={meta.error}
+          fieldName={field.name}
+        />
+      </VStack>
+    </FormControl>
+  );
+};
+
+export const CustomDateTimeInput = ({
+  label,
+  name,
+  tooltip,
+  disabled,
+  isRequired,
+  ...props
+}: CustomInputProps & FieldHookConfig<string>) => {
+  const [field, meta, { setValue, setTouched }] = useField(name);
+  const isInvalid = !!(meta.touched && meta.error);
+
+  const { validateField } = useFormikContext();
+
+  const fieldId = props.id ?? name;
```

Comment:
Is there ever a chance that props.id could be an empty string `""`? If so the `||` operator is likely safer to use here.
```suggestion
  const fieldId = props.id || name;
```